tests/batch/test_get_list_batches.py

```diff
@@ -0,0 +1,417 @@
+        '--reverse',
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
+
+def test_get_list_batches_by_head():
+    """
+    Case: get a list of batches by block identifier.
+    Expect: batches are returned.
+    """
+    head = '56100bf24eed12d2f72fe3c3ccf75fe2f53d87c224d9dda6fb98a1411070b06a' \
+           '40fcf97fccc61cb9c88442953af6ae50344ad7773f1becc6bae108443c18c551'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--head',
+        head,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
+
+@pytest.mark.parametrize('command_flag', ('--start', '--head'))
+def test_get_list_batches_with_invalid_start_head(command_flag):
+    """
+    Case: get a list of batches by invalid block identifier and batch identifier starting from.
+    Expect: the following identifier is invalid error message.
+    """
+    invalid_id = '044c7db163cf21ab9eafc9b267693e2d732411056c7530e54282946ec47cc180'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        command_flag,
+        invalid_id,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            f'{command_flag[2:]}': [
+                f'The following identifier `{invalid_id}` is invalid.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_limit():
+    """
+    Case: get a list of batches by limit.
+    Expect: batch is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--limit',
+        1,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert isinstance(json.loads(result.output), dict)
+
+
+def test_get_list_batches_with_invalid_limit():
+    """
+    Case: get a list of batches by invalid limit.
+    Expect: limit must be greater than 0 error message.
+    """
+    invalid_limit = -33
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'batch',
+        'get-list',
+        '--limit',
+        invalid_limit,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'limit': [
+                'Limit must be greater than 0.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_batches_with_invalid_node_url():
```

Comment:
This test is for a non-existing node URL, but not an invalid node URL. Update documentation.
```suggestion
def test_get_batch_status_with_non_existing_node_url():
```
Create test for invalid node URL.

Reply:
Done.